### PR TITLE
79 convert the test site banner into a generic notification banner

### PR DESCRIPTION
**Pull request by CaptNemoUK**
v0.1 of the new notification banner component.

Has 4 message types: error,warning,debug,info

Also has the option for a modal dialog box.

### CODE REVIEW

**Review by CapnFlint — CHANGES_REQUESTED** [review]
This needs to go into Dev, not Main.

**Comment by CaptNemoUK** [comment]
Changes committed to remote branch and pull request changed to merge to dev.

File: package-lock.json
Comment (CapnFlint) [inline package-lock.json:1]: This shouldn't be here. Please remove it and re-commit, thanks!

File: app/layouts/default.vue [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <template>
   <div class="app">
     <layout-header-default />
-    <panel class="warning-panel" v-if="config.public.test_env">
-      Test Version. Content May Not Persist.
-    </panel>
+
+    <widgets-notification messageType="warning" messageText="Test Version - Data May Not Persist" :modality="true" v-if="config.public.test_env" >
```
Comment (CapnFlint) [inline app/layouts/default.vue:5]: If we aren't using/needing the modal here, remove the content and set modality to false please.

**Review by CapnFlint — CHANGES_REQUESTED** [review]
Just a couple tweaks, but otherwise looks good!

**Review by CapnFlint — APPROVED** [review]
LGTM!

**Pull request merged**